buildKonfig.properties.example.properties

```diff
@@ -0,0 +1,13 @@
+# Staging env
+STAGING_CLIENT_ID=R9v5TscbY-_gxCtcmeJHIPm5tzXKkopGS**********
```

Comment:
Are they real data? 🙈

Reply:
Half of real is not real :D You can't use it or guess it. Just shed light on how to use this file.